src/test/java/api/requests/RequestsAPIRelatedRecordsTests.java

```diff
@@ -101,38 +108,43 @@ public void holdingAndInstanceIdComesFromMultipleRecordsForMultipleRequests()
 
     List<JsonObject> fetchedRequestsResponse = requestsClient.getAll();
 
-    JsonObject firstFetchedRequest = getRecordById(
-      fetchedRequestsResponse, firstRequestId).get();
-
-    JsonObject secondFetchedRequest = getRecordById(
-      fetchedRequestsResponse, secondRequestId).get();
+    JsonObject firstItem = getRecordById(fetchedRequestsResponse, firstRequestId).
```

Comment:
Please rename this test to better fit the increased scope of it to cover copy numbers